src/dashboard/pages/devices.py

```diff
@@ -16,9 +16,9 @@
                         _get_dropdown(
                             children=[
                                 dcc.Checklist(
-                                    id="num_devices_per_identity$checkbox",
+                                    id={"type": "hide-test-clients-checkbox", "plot": "num-devices-per-identity"},
                                     options=[{"label": "Hide Test Clients?", "value": "hide_test_clients"}],
-                                    value=["hide_test_clients"] if config.get().DASHBOARD_HIDE_TEST_CLIENTS_DEFAULT else [],
+                                    value=[],
```

Comment:
Benötigen wir das Einlesen der config hier noch? Können wir das nicht analog zu den anderen Plots gestalten?:
```suggestion
                                    value=[],
```